src/pages/Scheduling/ScheduleExceptions.tsx

```diff
@@ -63,6 +77,7 @@ const ScheduleExceptionItem = (
 ) => {
   const { t } = useTranslation();
   const queryClient = useQueryClient();
+  const [isAlertDialogOpen, setIsAlertDialogOpen] = useState(false);
```

Comment:
a component could potentially have multiple alert dialogs; improve namings

```suggestion
  const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
```

Reply:
👍